apps/tsconfig.json

```diff
@@ -13,7 +13,9 @@
 
     /* Language and Environment */
     "target": "es5" /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */,
-    // "lib": [],                                        /* Specify a set of bundled library declaration files that describe the target runtime environment. */
+    "lib": [
+      "ES2019"
```

Comment:
This was needed to use the `Array.flat()` function

Reply:
this looks okay, we just might have to be careful when we use ES2019 features to make sure they're supported by our browser matrix (or we polyfill or whatever to make it work). the [browser compatibility chart for Array.prototype.flat()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flat#browser_compatibility) looks like it supports all of our browsers in [browsers.json](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/ui/browsers.json), so we should be good, at least for now!